src/main/views/waveform.overview.js

```diff
@@ -183,6 +183,9 @@ define([
   };*/
 
   WaveformOverview.prototype.updateRefWaveform = function (time_in, time_out) {
+    if (isNaN(time_in))  throw new Error("WaveformOverview#updateRefWaveform passed a time in that is not a number: " + time_in);
```

Comment:
There _shouldn't_ be any reason why these will ever be NaN, but the bug was extremely subtle and I only found it by checking the arguments are the correct type. Since you're expecting time_in and time_out to be numbers, I am asserting that they are at the top of the method. I should be doing these assertions everywhere in waveform.*.js, so if I missed one, let me know and I'll add it :)